src/tests/test_bootstrap_in_place.py

```diff
@@ -279,6 +279,19 @@ def waiting_for_installation_completion(
             if not skip_logs:
                 self.log_collection(master_ip)
 
+    def inject_bootstrap(self, ignition_filename: string, butane_config: string):
```

Comment:
afaik it needs to be:
```suggestion
    def inject_bootstrap(self, ignition_filename: str, butane_config: str):
```